.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: node_js
+arch:
+  - amd64
+  - ppc64le
 node_js:
   - 'stable'
-  - '12'
```

Comment:
We absolutely can’t stop testing on these node versions.

Reply:
yes, can't stop on these versions .kindly merge the code with power  architecture.

Power architecture - primarily the bigger enterprise class systems from IBM for banking/insurance/retail/etc.etc but this ensures that your code and all dependencies are tested on the Power architecture. Debian/Ubuntu, and Fedora/RHEL/CentOS have ppc64le (Power) distributions and this code goes into those distributions. This ensures that top of tree is always validated on ppc64le using (free) Power servers at Travis CI.

Reply:
Code that doesn’t touch the filesystem, and isn’t using native modules, basically can’t vary by architecture - there’s no value in testing on it.

Reply:
For more info tag @gerrith3